### PR TITLE
Is this you

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
[Boost records with affiliations in the "Could this be you?" modal](https://trello.com/c/XrHvdPDG/14-boost-records-with-affiliations-in-the-could-this-be-you-modal)

File: src/app/register/pages/register/register.component.ts [hunk 1]
```diff
@@ -209,21 +212,23 @@ export class RegisterComponent implements OnInit, AfterViewInit {
 
     if (this.FormGroupStepA.valid) {
       this.personalData = this.FormGroupStepA.value.personal
+      const searchValue =
+        this.personalData.familyNames.value +
```
Comment (leomendoza123) [inline src/app/register/pages/register/register.component.ts:216]: if the user has no `familyNames`  is it ok to send the white space before the `givenName`?

File: src/app/register/pages/register/register.component.ts [hunk 2]
```diff
@@ -209,21 +212,23 @@ export class RegisterComponent implements OnInit, AfterViewInit {
...
+        ' ' +
+        this.personalData.givenNames.value
+      const searchParams: SearchParameters = {}
+      searchParams.searchQuery = searchValue
 
-      this._register
-        .checkDuplicatedResearcher({
```
Comment (leomendoza123) [inline src/app/register/pages/register/register.component.ts:214]: Maybe we can delete `checkDuplicatedResearcher`

File: src/app/register/pages/register/register.component.ts (hunk 1 above)
Comment (amontenegro) [reply src/app/register/pages/register/register.component.ts:216]: Hi @leomendoza123 yes, it is ok to send the space, the string get trimmed on server side.

File: src/app/register/pages/register/register.component.ts (hunk 2 above)
Comment (amontenegro) [reply src/app/register/pages/register/register.component.ts:214]: There is another card for cleanup @leomendoza123 https://trello.com/c/sDQrtQuI/781-remove-code-related-to-dupicateresearcherjson

File: src/app/cdk/is-this-you/is-this-you.component.html [hunk 3]
```diff
@@ -19,18 +19,20 @@ <h1 class="mat-display-1 modal-title">{{ data.titleLabel || titleLabel }}</h1>
...
         </th>
       </tr>
       <tr *ngFor="let record of data.duplicateRecords">
-        <td>{{ record.givenNames }}</td>
-        <td>{{ record.familyNames }}</td>
-        <td>{{ record.institution }}</td>
-        <td>{{ record.createdDate }}</td>
+        <td>{{ record['given-names'] }}</td>
+        <td>{{ record['family-names'] }}</td>
+        <td>{{ record['institution-name'].join(', ') }}</td>
```
Comment (DanielPalafox) [inline src/app/cdk/is-this-you/is-this-you.component.html:29]: Created date is missing

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**